lib/kubernetes_template_rendering/template.rb

```diff
@@ -24,9 +27,25 @@ def with_auto_generated_yaml_comment(yaml_string)
       comment = <<~EOS
       # WARNING: DO NO EDIT THIS FILE!
       # Any changes made here will be lost.
-      # This file is autogenerated from #{template_path}
+      # This file is autogenerated from #{full_template_path}
       EOS
-      comment + yaml_string
+      comment + variable_override_comment + yaml_string
+    end
+
+    def full_template_path
+      if source_repo
+        "https://github.com/#{source_repo}/blob/master/#{template_path.start_with?("/") ? template_path[1..-1] : template_path}"
```

Comment:
⛏️ Github allows the use of `-` to mean "the default branch" of repositories, might want to use that here.

Examples:
* https://github.com/Invoca/invocaops_docker/blob/-/jsonnet-libraries/main.libsonnet
* https://github.com/Invoca/invoca-backstage/blob/-/README.md

Reply:
I knew there was a syntax for this, but couldn't remember, thank you! Done fb38d6171d0906b8e7eaedbc81502ca8a12c1704